docs/stock/understanding-highcharts-stock.md

```diff
@@ -5,30 +5,92 @@ Highcharts Stock is based on Highcharts, meaning it has all the core functionali
 
 ![understanding_highstock.png](understanding_highstock.png)
 
-Highcharts Stock also supports candlestick and OHLC charts.
+Highcharts Stock also supports various financial series types.
 
-Navigator
+<iframe style="width: 100%; height: 550px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/series-type allow="fullscreen"></iframe>
```

Comment:
Q: No need for the quotes in `src`? 🤔

Reply:
Interesting 🤔 Looks like it works without problems on master https://github.com/highcharts/highcharts/blob/3e84c2f0ec8585088a97bb1aebef8fcbd60caa95/docs/stock/axis-resizer.md?plain=1#L24